hardware_interface/src/fake_components/generic_system.cpp

```diff
@@ -452,6 +452,16 @@ void GenericSystem::initialize_storage_vectors(
               "The usage of initial_%s has been deprecated. Please use 'initial_value' instead.",
               interface.name.c_str());
           }
+          else
+          {
+            RCUTILS_LOG_WARN_ONCE_NAMED(
+              "fake_generic_system",
+              "Parsing of optional initial interface values failed. Add initial values for every "
+              "state interface in the ros2_control.xacro like: \n"
+              "<state_interface name=\"position\"> \n"
```

Comment:
"Position" is misleading here. Every interface type may have initial value.

Reply:
Position is just an example. I think that's clear enough.

Overall, I think adding this hint would be a good way to help people start using this feature. Right now, the only way they would know to use `initial_value` is if they happen to see your PR.

On the other hand, it's a very minor feature and I don't intend to spend much more time on it.

Reply:
OK, I improved the original error message a bit and it should definitely only print once per ros_control xacro now.